app/Libraries/Elasticsearch/Indexing.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace App\Libraries\Elasticsearch;
+
+use Elasticsearch\Common\Exceptions\Missing404Exception;
+use Es;
+
+class Indexing
+{
+    public static function deleteIndex(string $name)
+    {
+        return Es::indices()->delete([
+            'index' => $name,
+            'client' => ['ignore' => 404],
+        ]);
+    }
+
+    public static function getOldIndices(string $alias)
```

Comment:
it's just one `OldIndex`, isn't it?

Reply:
wut it's not ಠ_ಠ

Reply:
alias can have many indices and index can have many aliases :D